api/resource_releases.go

```diff
@@ -67,3 +105,31 @@ func (c *Client) GetAppRelease(ctx context.Context, appName string, id string) (
 
 	return data.App.Release, nil
 }
+
+func (c *Client) GetAppReleaseMachines(ctx context.Context, appName string, id string) (*Release, error) {
+	query := `
+		query ($appName: String!, $releaseId: ID!) {
+			app(name: $appName) {
+				release: releaseUnprocessed(id: $releaseId) {
```

Comment:
does this one work? I don't think I wired that up. it would be easy to do so.

Reply:
d'oh, you're right. I saw the data type in the graphql schema update, then the symmetry in the request code and it never even occurred to me.

you could wire it up or I could just remove this code path for now. it's unused so either's fine imo

I think it might be wise to just hook it up, but if that's more than seriously like a 15 minute job then maybe that's unnecessary

Reply:
oh if it's not used, how about removing from here? thx!